<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Startup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="defaultConfigurationCheckBox">
            <property name="text">
             <string>Use default configuration</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>24</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="defaultConfigurationLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="defaultConfigurationButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Choose...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Event</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="openEventEditorCheckBox">
            <property name="text">
             <string>Open the editor when inserting an event</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Ground orientation</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Default orientation for the automatic detection:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="defaultPlaneOrientationComboBox">
            <item>
             <property name="text">
              <string>Plane XY</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Plane YZ</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Plane ZX</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>150</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Time bar event</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Default display:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="defaultTimeBarEventDisplayComboBox">
            <item>
             <property name="text">
              <string>Frame</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Time</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_23">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>7</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_29">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>(also affect the X axis of the chart)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="visualisationTab">
      <attribute name="title">
       <string>Visualisation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Background</string>
         </property>
         <layout class="QFormLayout" name="formLayout_6">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Background color:             </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="defaultBackgroundColorButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
  border: 1px solid darkgray;
  background: rgb(0,0,0);
}

QPushButton:pressed {
  border: 1px solid rgb(100,100,100);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Grid color (front &amp; back):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="defaultGridFrontColorButton">
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid darkgray;
  background: rgb(204,204,204);
}

QPushButton:pressed {
  border: 1px solid rgb(100,100,100);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="defaultGridBackColorButton">
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid darkgray;
  background: rgb(255,0,0);
}

QPushButton:pressed {
  border: 1px solid rgb(100,100,100);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Segment</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="defaultSegmentColorLabel">
            <property name="text">
             <string>Default color:                    </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="defaultSegmentColorButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
  border: 1px solid darkgray;
  background: white;
}

QPushButton:pressed {
  border: 1px solid rgb(100,100,100);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Marker</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="defaultMarkerColorLabel">
            <property name="text">
             <string>Default color:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="defaultMarkerColorButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
  border: 1px solid darkgray;
  background: white;
}

QPushButton:pressed {
  border: 1px solid rgb(100,100,100);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="defaultMarkerRadiusLabel">
            <property name="text">
             <string>Default radius:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="defaultMarkerRadiusSpinBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="defaultMarkerTrajectoryLengthLabel">
            <property name="text">
             <string>Default trajectory length:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="defaultMarkerTrajectoryLengthComboBox">
            <item>
             <property name="text">
              <string>All Frames</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 Frames</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 Frames</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100 Frames</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200 Frames</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Force platform</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="showForcePlatformAxesLabel">
            <property name="text">
             <string>Show axes:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="showForcePlatformAxesComboBox">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="showForcePlatformIndexLabel">
            <property name="text">
             <string>Show index:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="showForcePlatformIndexComboBox">
            <item>
             <property name="text">
              <string>Yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="defaultForcePlateColorLabel">
            <property name="text">
             <string>Force plate color:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="defaultForcePlateColorButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
  border: 1px solid darkgray;
  background: white;
}

QPushButton:pressed {
  border: 1px solid rgb(100,100,100);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="defaultForceVectorColorLabel">
            <property name="text">
             <string>Force vector color:           </string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="defaultForceVectorColorButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
  border: 1px solid darkgray;
  background: white;
}

QPushButton:pressed {
  border: 1px solid rgb(100,100,100);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="defaultGRFButterflyActivationLabel">
            <property name="toolTip">
             <string>Display ground reaction forces (GRF) butterfly diagram</string>
            </property>
            <property name="text">
             <string>Butterfly diagram:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="defaultGRFButterflyActivationComboBox">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="showForcePathLabel">
            <property name="text">
             <string>Force path:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="showForcePathComboBox">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="chartTab">
      <attribute name="title">
       <string>Chart</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Plot</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Default line width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="defaultPlotLineWidthSpinBox">
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Graph</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Display events:    </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="defaultChartEventDisplayComboBox">
            <item>
             <property name="text">
              <string>Yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Axis X unit:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="defaultChartUnitAxisXComboBox">
            <item>
             <property name="text">
              <string>Frame</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Time</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>265</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutsTab">
      <attribute name="title">
       <string>Layouts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Layout Manager</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="LayoutsTableWidget" name="layoutTable"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Drag and drop the layouts to reorder them.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Double click on a layout to rename it.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Update</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QCheckBox" name="automaticCheckUpdateCheckBox">
            <property name="text">
             <string>Automatically check for update at startup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="subscribeDevelopmentChannelCheckBox">
            <property name="text">
             <string>Subscribe to the development channel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>235</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayoutsTableWidget</class>
   <extends>QTableWidget</extends>
   <header>Preferences_Layouts.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultConfigurationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>defaultConfigurationLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultConfigurationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>defaultConfigurationButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
